x/evm/types/params_test.go

```diff
@@ -127,3 +181,146 @@ func TestIsLondon(t *testing.T) {
 		require.Equal(t, IsLondon(ethConfig, tc.height), tc.result)
 	}
 }
+
+func TestCheckIfEnabledPrecompilesAreRegistered(t *testing.T) {
+	m := func(addr string) precompile_modules.Module {
+		return precompile_modules.Module{
+			Address: common.HexToAddress(addr),
+		}
+	}
+	a := func(addr string) string {
+		return common.HexToAddress(addr).String()
+	}
+
+	testCases := []struct {
+		name               string
+		registeredModules  []precompile_modules.Module
+		enabledPrecompiles []string
+		expError           bool
+	}{
+		{
+			name:               "test-case #1",
```

Comment:
some descriptive test names instead of numbers would be useful 😄 Can use `t.Run(tc.name, func(t *testing.T) {` in the loop too

Reply:
Fixed: https://github.com/Kava-Labs/ethermint/pull/55/commits/770e0bad937b9bafdf085e546b678d75cc803839